ntcore/src/generate/java/NetworkTablesJNI.java.jinja

```diff
@@ -262,6 +262,8 @@ public final class NetworkTablesJNI {
 
   public static native boolean isConnected(int inst);
 
+  public static native Long getServerTimeOffset(int inst);
```

Comment:
Any reason this is a boxed `Long` rather than `long`?

Reply:
I need to return an optional value.  Since OptionalLong is a thing, I'll use that instead.